### PR TITLE
scrape cert-manger and fluent-bit metrics targets

### PR DESCRIPTION
**Pull request by zjj2wry**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes # https://github.com/tidbcloud/infra/issues/524 https://github.com/tidbcloud/infra/issues/517

**Special notes for your reviewer**:

**Release note**:

```improvement operator

```

### CODE REVIEW

**Review by qiffang — APPROVED** [review]
lgtm

File: charts/seed-monitoring/charts/core/charts/prometheus/templates/config.yaml [hunk 1]
```diff
@@ -86,6 +86,159 @@ data:
...
+      - source_labels: [__meta_kubernetes_service_label_cert_manager]
+        separator: ;
+        regex: (.+)
+        target_label: job
+        replacement: ${1}
+        action: replace
+      - separator: ;
+        regex: (.*)
+        target_label: endpoint
+        replacement: "9402"
```
Comment (aylei) [inline charts/seed-monitoring/charts/core/charts/prometheus/templates/config.yaml:168]: why do we hardcode here?

Comment (zjj2wry) [reply charts/seed-monitoring/charts/core/charts/prometheus/templates/config.yaml:168]:
```
spec:
  clusterIP: 172.20.78.166
  ports:
  - port: 9402
    protocol: TCP
    targetPort: 9402
  selector:
    app.kubernetes.io/component: controller
    app.kubernetes.io/instance: cert-manager
    app.kubernetes.io/name: cert-manager
  sessionAffinity: None
  type: ClusterIP
```

cert-manager service does not set port name

**Review by aylei — APPROVED** [review]
lgtm

**Pull request merged**